File: 2137-FinalValueOfVariableAfterPerformingOperations/2137-FinalValueOfVariableAfterPerformingOperations.java
// Last updated: 9/3/2025, 10:47:05 AM
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int X=0;
        for(String i: operations){
            if(i.equals("X++") || i.equals("++X")){
                X++;
            }
            else if(i.equals("X--") || i.equals("--X")){
                X--;
            }
        }
        
        return X;

    }
}

File: 2137-FinalValueOfVariableAfterPerformingOperations/2137-FinalValueOfVariableAfterPerformingOperations.py
# Last updated: 9/3/2025, 10:47:00 AM
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
       X = 0
       for i in operations:
        if(i in ("X++","++X")):
            X+=1
        elif(i in ("X--","--X")):
            X-=1
       return X
        
        